lib/osf-components/addon/components/file-browser/add-new/upload/component.ts

```diff
@@ -28,6 +28,7 @@ export default class Upload extends Component<Args> {
         autoProcessQueue: true,
         autoQueue: true,
         parallelUploads: this.args.manager.currentFolder.parallelUploadsLimit,
+        maxFilesize: 10000000,
```

Comment:
so I tried adding a file larger than 10MB (1GB), and it didn't give any indication that that wasn't allowed. Is there a friendly way to tell users they can't do that? At the least, we could tell them we don't allow files >10MB at the top of the modal and/or in the help modal

Reply:
The `maxFilesize` is in mb. So it is acutally 10000 gb.

Reply:
I am pretty sure it is not in bytes because I can clearly upload stuff more than 10000000 bytes. Here is the exact line where it is checking for the size: https://github.com/dropzone/dropzone/blob/v5.7.3/src/dropzone.js#L744

Reply:
That's interesting. In that case, sorry to bother you with my local-env problem!